http-service-lambda/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+authors = ["Kan-Ru Chen <kanru@kanru.info>"]
+description = "HttpService server that AWS Lambda Rust Runtime as backend"
+documentation = "https://docs.rs/http-service-lambda"
+edition = "2018"
+license = "MIT OR Apache-2.0"
+name = "http-service-lambda"
+repository = "https://github.com/rustasync/http-service"
+version = "0.1.0"
+
+[dependencies]
+http-service = "0.2.0"
+lambda_http = "0.1.1"
+lambda_runtime = "0.2.1"
+tokio = "0.1.21"
+
+[dependencies.futures-preview]
+features = ["compat"]
+version = "0.3.0-alpha.16"
+
+[dev-dependencies]
+log = "0.4.6"
+simple_logger = { version = "1.3.0", default-features = false }
+tide = "0.2.0"
```

Comment:
It's common for components that rely on Tide to [depend on Git `master` in development](https://github.com/rustasync/tide/blob/master/tide-compression/Cargo.toml#L30). We might want to adopt that practice here as well.

Reply:
This sounds a good idea, but I'm not sure how to solve the `http-service` dependency. Currently I have

```toml
[dependencies]
http-service = "0.2.0"
```

But I would like to use

```toml
[dependencies]
http-service = { version = "0.2.0", path = ".." }
```

It creates a conflict between Tide because Tide wants to use the git version instead of the local version:

```sh
% cargo tree -i -p https://github.com/rust-lang/crates.io-index#http-service:0.2.0
http-service v0.2.0
├── http-service-hyper v0.2.0
│   └── tide v0.2.0 (https://github.com/rustasync/tide#25de4138)
│       [dev-dependencies]
│       └── http-service-lambda v0.1.0 (/home/kanru/r/http-service/http-service-lambda)

% cargo tree -i -p file:///home/kanru/r/http-service#0.2.0
http-service v0.2.0 (/home/kanru/r/http-service)
└── http-service-lambda v0.1.0 (/home/kanru/r/http-service/http-service-lambda)
```

Reply:
Sounds like an issue we may need to patch up elsewhere. Thanks for sharing the issues you encountered. I'm not sure how to resolve this error either, but I'll do some research and see if a workable solution can be found.

Reply:
Today I learned this has to be overridden in the main `Cargo.toml` for a workspace. Leave this as is, it's possible to update to better practices in a separate PR.